### DISCUSSION TITLE
Ray tracing for indoor scenarios

### DISCUSSION THREAD

**Discussion by jasperbrg** [NVlabs/sionna#334] [General]
Hi everyone,

As part of an aeronautical project, I am exploring Sionna ray tracing to model the channel behavior inside of an aircraft cabin. However, since this is quite a confined space with a large number of small scatterers with often rather small conductivity, I expect that the current state of Sionna will be rather limited in terms of achievable modeling accuracy. This will probably be mainly due to the lack of refraction computation in the ray tracer. 

This leads to my question, whether we can expect to see a refraction feature in the near future?

Thanks in advance!

**Comment by jhoydis** [comment]
Hi,

Refraction is unlikely to come to Sionna RT in the next couple of months.
Then there would also be the problem of modelling humans, seats, etc, which have all very complex shapes and behaviour.
So, I would not use Sionna RT for modeling of such complex indoor scenes for now.

**Reply by alialsahlany** [reply]
Did the refraction feature add to Shionna?

**Reply by merlinND** [reply]
Hello @alialsahlany,

Refraction is not currently supported in Sionna.
